src/routes/reset-password/containers/ResetPasswordPage.js

```diff
@@ -0,0 +1,14 @@
+const { useMiddleEnd } = require('strange-middle-end');
+const ResetPasswordPage = require('../components/ResetPasswordPage');
+const GetQueryParams = require('../../../utils/get-query-params');
+
+module.exports = function ResetPasswordPageContainer(props) {
+
+    const m = useMiddleEnd();
+
+    const { user: userHash, token: forgotPasswordToken } = GetQueryParams();
```

Comment:
Could we lean on the `useParams` hook here? https://v5.reactrouter.com/web/api/Hooks/useparams